astropy/conftest.py

```diff
@@ -63,6 +65,12 @@ def fast_thread_switching():
     sys.setswitchinterval(old)
 
 
+@pytest.fixture
+def rng():
```

Comment:
`rng` is also too cryptic for a fixture that would be used package-wide. Maybe something a bit more descriptive?

Reply:
how about `random_number_generator` ?

Reply:
If people think that is too long, `random_num_gen` might also work for me but I cannot speak for everyone.

Reply:
IDK. Anyone using Astropy is using Numpy (/ an Array API library in future) and tHat's the standard name.

Reply:
I think we're leaning towards it is not necessary at all.